repos.yaml

```diff
@@ -126,7 +126,7 @@
   - encoding/simplifiedchinese
   - encoding/traditionalchinese
   - encoding/unicode
-  - encoding/utf32
```

Comment:
I believe this was wrong?

Reply:
Good catch!  We could maybe have a  `-verify`  that just checks the repos are present in the corpus if the repos exists or something.